markdown/usage/installation.md

```diff
@@ -146,6 +146,18 @@ nf-core pipelines utilise the built-in support for software packaging that Nextf
   - Poorer reproducibility than Docker / Singularity
     - There can be changes in low-level package dependencies over time
     - The software still runs in your native operating system environment and so core system functions can differ
+- [Mamba](https://mamba.readthedocs.io/)
+  - A faster reimplementation of Conda
+  - Packaging system that manages environments instead of running analysis in containers
+  - Poorer reproducibility than Docker / Singularity
+    - There can be changes in low-level package dependencies over time
+    - The software still runs in your native operating system environment and so core system functions can differ
+- [Apptainer](https://apptainer.org/)
```

Comment:
I would slightly prefer to place the Apptainer section right after the Singularity one.

Furthermore, we could refer to the Nextflow documentation, e.g. with a Note, Box or Blockquote in Markdown?

> ``📝``  _Please refer to the [Nextflow documentation](https://nextflow.io/docs/edge/container.html#apptainer) on how to configure the system correctly._